Assets/Scripts/DataTypes/Car.cs

```diff
@@ -1,33 +1,49 @@
 using UnityEngine;
 
 namespace DataTypes
-{ 
-    public class Car
+{
+    public class Car : GameObjectData<Car, CarBehaviour>
     {
         public Edge road { get; private set; }
         public float positionOnRoad { get; private set; }
         public float lane { get; private set; }
         public float speed { get; private set; } = Conversion.UnitsPerTimeStepFromKPH(50); // Laengeneinheiten pro Zeiteinheit
-        public Transform carTransform;
 
-        public Car(Edge road, float positionOnRoad, float lane)
+        public Car(GameObject prefab, Edge road, float positionOnRoad, float lane)
         {
             this.road = road;
             this.road.cars.Add(this);
             this.positionOnRoad = positionOnRoad;
             this.lane = lane;
+            
+            // display the car graphically
+            var position = road.GetAbsolutePosition(positionOnRoad, lane);
+            CreateGameObject(
+                prefab: prefab,
+                position: new Vector3(position.x, prefab.transform.localScale.y / 2 + prefab.transform.localScale.y / 2, position.y),
+                rotation: Quaternion.Euler(0, road.angle, 0)
+            );
+            gameObject.name = "Car_" + CarId.id;
         }
-       
+
         public void Move()
         {
             positionOnRoad += speed;
             var position = road.GetAbsolutePosition(positionOnRoad, lane);
-            carTransform.position = new Vector3(position.x, carTransform.position.y, position.y);
+            gameObject.transform.position = new Vector3(position.x, gameObject.transform.position.y, position.y);
         }
 
         public void Accelerate(float acceleration)
         {
             speed += acceleration;
         }
     }
+
+    public class CarBehaviour : LinkedBehaviour<Car>
+    {
+        private void FixedUpdate()
+        {
+            _data.Move();
+        }
```

Comment:
Ich dachte, wir hätten uns darauf geeinigt, dass die Move-Funktion zentral aufgerufen wird, um zu gewährleisten, dass alle Autos sich zeitgleich bewegen. Siehe Diskussion in #7

Reply:
naja wir werden ja nun sphären nutzen, um die moves zu planen, wobei die kollisionen ja automatisch im physicsupdate gecheckt werden, welches vor dem fixedupdate geschieht

Reply:
Wie hier zu sehen ist, ist das interne physicsupdate nach fixedUpdate: https://docs.unity3d.com/Manual/ExecutionOrder.html

Reply:
...was außer für das erste update aufs gleiche hinausläuft.

Reply:
Touché